class.jetpack.php

```diff
@@ -3872,6 +3876,31 @@ function disconnect_survey_notice() {
 		<?php
 	}
 
+	/**
+	 * Checks if the user have started a register flow from calypso in the last day, and if so, redirect them there
+	 */
+	function check_jetpack_connect_redirection() {
+		$user = $this->get_connected_user_data();
+		$parsed_site = parse_url( get_site_url() );
+
+		if ( isset( $user[ 'jetpack_connect' ] ) && is_array( $user[ 'jetpack_connect' ] ) ) {
```

Comment:
WordPress PHP spacing code style for array items are like so: 

`$var[ $var ]` (with spaces)
`$var['string']` (no spaces)